src/gui/qjacktrip.cpp

```diff
@@ -210,6 +214,46 @@ QJackTrip::QJackTrip(QWidget *parent) :
     } else {
         m_autoQueueIndicator.setText("Auto queue: disabled");
     }
+    
+#ifdef USE_WEAK_JACK
+//Check if Jack is actually available
+    if (have_libjack() != 0) {
+#ifdef __RT_AUDIO__
```

Comment:
IUUC correctly this only supports weakjack + rtaudio configuration, correct? While I don't think we plan to ship that, from a configuration standpoint it might be prudent to allow jack + rtaudio config as well (jack, not weakjack)

Reply:
I'm thinking about jack + rtaudio for flatpaks to use the pulseaudio backend.

Reply:
This is to detect the situation where libjack isn't available on the system and can't be dynamically loaded by weakjack so the jack backend is unusable. All of my builds are dynamic, so the test you're suggesting would never happen without weakjack because of a segfault. Is the jack library itself being statically linked in the static builds?

Reply:
You can't (or you shouldn't) link jack statically. Your code looks good to me.

Reply:
Yeah, that's what I thought. So we can only test for this situation if we're using weakjack.

Reply:
Got it. I originally thought that some of that ifdef guargs RtAudio-only functionality (which we want to maintain even if we're not using weakjack) but that wasn't correct.

Just to clarify one point:
I think @psiborg112 meant _loading_ jack dynamically, right? Sure, we shouldn't link jack statically, but when it's _linked_ dynamically (not loaded dynamically, i.e. not using weakjack) when the app wouldn't even start if jack is not present.